Code/Source/RobotControl/ROS2RobotControlComponent.h

```diff
@@ -21,6 +21,13 @@ namespace ROS2
     {
     public:
         AZ_COMPONENT(ROS2RobotControlComponent, "{CBFB0764-99F9-40EE-9FEE-F5F5A66E59D2}", AZ::Component);
+        ROS2RobotControlComponent()
+        {
+        }
+        ROS2RobotControlComponent(ControlConfiguration controlConfiguration)
+            : m_controlConfiguration(controlConfiguration)
```

Comment:
The Controlconfiguration object can be moved here.
```suggestion
            : m_controlConfiguration(AZStd::move(controlConfiguration))
```